### PR TITLE
Better error handling with sql.Rows

### PR DESCRIPTION
**Pull request by tolikzinovyev**

### CODE REVIEW

File: idb/postgres/postgres.go [hunk 1]
```diff
@@ -1758,6 +1766,18 @@ func (db *IndexerDb) yieldTxnsThreadSimple(ctx context.Context, rows *sql.Rows,
 			count++
 		}
 	}
+	if err := rows.Err(); err != nil {
+		select {
```
Comment (winder) [inline idb/postgres/postgres.go:1770]: No need for a select here, just set errp if it hasn't already been set.

**Review by winder — CHANGES_REQUESTED** [review]
Looks good, one minor comment

**Review by brianolson — APPROVED** [review]
good catch. this is definitely more proper error checking

**Comment by tolikzinovyev** [comment]
Thanks!

**Pull request merged**